routes/api/users.js

```diff
@@ -64,4 +64,14 @@ router.put(
 // The Route to get the user profile
 router.get('/users/profile/:username', userControllers.getProfile);
 
+// super-admin and admin get all users
+router.get('/users', verifyAdmin, userControllers.getAllUsers);
+
+// super-admin and admin delete users
+router.delete('/users/:username', verifyToken, userControllers.deleteUser);
```

Comment:
@shemaeric  This should be **_verifyAdmin_**  and in the controller, there is code checking if the user is an admin or super admin.

Reply:
Same as on line 73 and 74